servicebroker/lib/util/apiServerUtil.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+var request = require('request');
+
+module.exports = function(apiServerUri){
+  var apiUtil = {};
+  apiServerUri = apiServerUri.replace(/\/$/g, "").toLowerCase();
```

Comment:
There are things like url-join but I get it. Might want to add a comment for the next person.